src/store/initStore.js

```diff
@@ -81,6 +83,32 @@ function addConfigToStore(store, config) {
   if (config.LOC) {
     store.dispatch(actions.initLoc(config.LOC));
   }
+  const { locFormId, country } = config.APP.mapConfig;
+  /**
+   * Check  location formId and country
+   * Get all locations and filter by country
+   */
+  if (country && locFormId) {
+    let q = d3.queue();
+    q = q.defer(getData, locFormId, null, config.APP);
```

Comment:
@kahama94 can we refactor to use `fetch()` method instead of d3 when making requests?